gulpfile.js

```diff
@@ -361,6 +361,96 @@ gulp.task("sonarqube:scan", async () => {
   }
 });
 
+/** CI ********************************************************************************************/
+
+// Replace each vss-extension.json version with the current build number
+gulp.task("ci:azure:hotfix-extensions-version", () => {
+  const buildNumber = process.env.BUILD_BUILDID;
+
+  if (!buildNumber) {
+    throw new Error("Missing build number");
+  }
+
+  const vssExtensions = globby.sync([
+    path.join(SOURCE_DIR, "extensions", "*", "vss-extension.json"),
+  ]);
+
+  return mergeStream(
+    vssExtensions.map((vssExtension) =>
+      gulp
+        .src(vssExtension)
+        .pipe(
+          gulpJsonEditor((json) => ({
+            ...json,
+            version: `${json.version}.${buildNumber}`,
+          })),
+        )
+        .pipe(gulp.dest(path.dirname(vssExtension))),
+    ),
+  );
+});
+
+gulp.task("ci:azure:hotfix-tasks-version", () => {
+  const buildNumber = process.env.BUILD_BUILDID;
+
+  if (!buildNumber) {
+    throw new Error("Missing build number");
+  }
+
+  const tasks = globby.sync(["src/extensions/*/tasks/*/v*/*.json"]);
+
+  return mergeStream(
+    tasks.map((task) =>
+      gulp
+        .src(task)
+        .pipe(
+          gulpJsonEditor((json) => ({
+            ...json,
+            version: {
+              ...json.version,
+              Patch: parseInt(buildNumber, 10),
```

Comment:
I'm not familiar with `Patch` in the version object. I assume it appends the Patch version to equal the build number

Reply:
Yep, it replaces [this `Patch`](https://github.com/SonarSource/sonar-scanner-vsts/blob/master/src/extensions/sonarqube/tasks/SonarQubePrepare/v5/task.json#L13) with the build number